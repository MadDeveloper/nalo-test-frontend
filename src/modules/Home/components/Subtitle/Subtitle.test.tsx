import renderer from "react-test-renderer"
import { Subtitle } from "."

describe("Home module > Subtitle", () => {
  it("should match snapshot", () => {
    const tree = renderer.create(<Subtitle>Lorem ipsum</Subtitle>).toJSON()

    expect(tree).toMatchSnapshot()
  })
})
