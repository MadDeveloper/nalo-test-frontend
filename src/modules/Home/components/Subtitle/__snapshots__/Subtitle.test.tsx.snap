// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home module > Subtitle should match snapshot 1`] = `
<h2
  className="root"
>
  Lorem ipsum
</h2>
`;
